src/main/resources/assets/numismatic-overhaul/lang/uk_ua.json

```diff
@@ -0,0 +1,69 @@
+{
+  "item.numismatic-overhaul.bronze_coin": "Бронзова монета",
+  "item.numismatic-overhaul.silver_coin": "Срібна монета",
+  "item.numismatic-overhaul.gold_coin": "Золота монета",
+  "item.numismatic-overhaul.money_bag": "Мішок із грішми",
+  "block.numismatic-overhaul.shop": "Крамниця",
+  "block.numismatic-overhaul.inexhaustible_shop": "Невичерпна крамниця",
+  "block.numismatic-overhaul.inexhaustible_shop.tooltip": "Лише в творчості — ніколи не вичерпуються запаси",
+  "block.numismatic-overhaul.piggy_bank": "Свинка-скарбничка",
+  "gui.numismatic-overhaul.shop.price": "Ціна",
+  "gui.numismatic-overhaul.shop.inventory_title": "Інвентар крамниці",
+  "gui.numismatic-overhaul.shop.merchant_title": "Крамниця",
+  "gui.numismatic-overhaul.shop.transfer_tooltip.enabled": [
+    "Hoppers",
+    {
+      "text": " can ",
+      "color": "#28FFBF"
+    },
+    "вставте \nпредметів у цю крамницю"
+  ],
+  "gui.numismatic-overhaul.shop.transfer_tooltip.disabled": [
+    "Hoppers",
+    {
+      "text": " cannot ",
+      "color": "#EB1D36"
+    },
+    "вставте \nпредметів у цю крамницю"
+  ],
```

Comment:
why are these keys not fully translated?

Reply:
This is the first time I've come across such a line, so I didn't understand what to translate from it

Reply:
This tooltip is for the button inside the shop which can enable/disable transferring items in/out of the shop block by hopper. An example would be having chests hopper into it for extra stock.

Reply:
https://docs.wispforest.io/owo/rich-translations/ is a good guide for this